packages/node-client/lib/index.ts

```diff
@@ -834,11 +834,18 @@ export class Nango {
         };
 
         if (config.params) {
-            options.params = config.params;
-        }
-
-        if (config.paramsSerializer) {
-            options.paramsSerializer = config.paramsSerializer;
+            if (typeof config.params === 'string') {
+                if (url.includes('?')) {
+                    throw new Error('Can not set query params in endpoint and in params');
+                }
+                url = new URL(`${url}${config.params.startsWith('?') ? config.params : `?${config.params}`}`).href;
+            } else {
```

Comment:
can `config.params` be neither a string nr an object?

Reply:
I guess it can if they don't respect the types. I would trust the types until we talk about adding runtime type checking